src/collective/cover/tests/test_layout_resize.robot

```diff
@@ -0,0 +1,116 @@
+*** Settings ***
+
+Resource  cover.robot
+Library  Remote  ${PLONE_URL}/RobotRemote
+
+Suite Setup  Open Test Browser
+Suite Teardown  Close all browsers
+
+
+
+*** Variables ***
+
+${basic_tile_name} =  "collective.cover.basic"
+${tile_class} =  div.cover-tile
+
+*** Test cases ***
+
+Test Columns and Titles
+    Set Selenium Speed  0.25 seconds
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+
+    Create Cover  Save-Cover  Test-cover
+    Open Layout Tab
+
+    Add Tile  ${basic_tile_name}
+    Click Element  css=i.resizer
+    Wait until element is visible  id=ui-id-1
+    Drag And Drop By Offset  css=a.ui-slider-handle.ui-state-default.ui-corner-all  -30  0
```

Comment:
After resizing the column, you should test that the `x` on text `Actual column size: x` has changed.
You should also test that the data-column-size attribute in `div.cover-column` has changed.

Reply:
After resizing the column, you should test that the x on text Actual column size: x has changed.

Reply:
This solution, I think, is more elegant, take the size you have and just check if "changing" really change.

gracias por el "feedback" :D @idgserpro

Reply:
If you resize in master branch, you will see that the `data-column-size` attribute changes. The problem this PR is fixing is that the text `Actual column size: x` is not changed. That's why I think it's important to do this test.

Reply:
@idgserpro  Ready :D!